build.gradle

```diff
@@ -2,10 +2,10 @@ apply plugin: 'com.vanniktech.code.quality.tools'
 
 buildscript {
     ext.versions = [
-            kotlin: '1.3.41',
-            minSdk: 16,
-            compileSdk: 28,
-            androidTools: '26.5.0',
+            kotlin: '1.3.72',
```

Comment:
We have later versions than this. Is this intentional for backward compatibility reasons?

Reply:
updated